language/move-vm/runtime/src/data_cache.rs

```diff
@@ -197,7 +199,20 @@ impl<'r, 'l, C: RemoteCache> DataStore for TransactionDataCache<'r, 'l, C> {
 
                     GlobalValue::cached(val)?
                 }
-                None => GlobalValue::none(),
+                Ok(None) => GlobalValue::none(),
+                Err(err) => {
+                    println!("HHIIIIIT");
```

Comment:
remove? 😉

Reply:
Yeah will get it removed. Agreed it's a good idea to check if there are any `println`s we forgot to delete.